libs/design/switch/src/switch/switch.component.ts

```diff
@@ -1,130 +1,95 @@
+/* eslint-disable quote-props */
 import { coerceBooleanProperty } from '@angular/cdk/coercion';
 import {
   ChangeDetectionStrategy,
   Component,
   EventEmitter,
-  HostBinding,
-  HostListener,
   Input,
   Output,
 } from '@angular/core';
 
-import { DaffDisableableDirective } from '@daffodil/design';
-import { DAFF_LOADING_ICON_COMPONENTS } from '@daffodil/design/loading-icon';
+import {
+  DaffDisableableDirective,
+  DaffSizableDirective,
+  DaffSizeSmallType,
+} from '@daffodil/design';
 
 import {
-  DaffLabelPosition,
-  DaffLabelPositionEnum,
+  DaffSwitchLabelPosition,
+  DaffSwitchLabelPositionEnum,
 } from './label-position';
 
-let switchUniqueId = 0;
+let switchUniqueLabelId = 0;
```

Comment:
Why two variables?

Reply:
There's a `labelId` and a `toggleId`.

`labelId` is set as the `id` for `<label>`, which handles the `aria-labelledby` on the button.

`toggleId` is set as the `id` for the button, which handles the `for` on `<label>`.